src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -692,8 +692,8 @@ public static Task RunImpersonatedAsync(SafeAccessTokenHandle safeAccessTokenHan
         /// </summary>
         /// <typeparam name="T">The type of the object to return.</typeparam>
         /// <param name="safeAccessTokenHandle">The SafeAccessTokenHandle of the impersonated Windows identity.</param>
-        /// <param name="func">The System.Func<System.Threading.Tasks.Task<T>> to run.</param>
-        /// <returns>A <see cref="Task{T}"/> that represents the asynchronous operation of the provided System.Func<System.Threading.Tasks.Task<T>>.</returns>
+        /// <param name="func">The <see cref="System.Func{Task}"/> of <see cref="System.Threading.Tasks.Task{T}"/> to run.</param>
+        /// <returns>A <see cref="Task{T}"/> that represents the asynchronous operation of the <see cref="System.Func{Task}"/> of <see cref="System.Threading.Tasks.Task{T}"/> provided.</returns>
```

Comment:
@stephentoub sorry for the commits above, I tried to update with github web(but without vs hint is hard) to avoid to recompile(this fork is a lot behind master).
Since nested generics isn't supported I used "extended" form, let me know if you've better idea or if there is a standard on it.